src/Frontend/Modules/Search/Ajax/Livesuggest.php

```diff
@@ -200,8 +202,11 @@ private function getRealData()
 
         // redirect if the request page doesn't exist
         if ($this->requestedPage > $this->pagination['num_pages'] || $this->requestedPage < 1) {
-            \SpoonHTTP::redirect(
-                FrontendNavigation::getURL(404)
+            throw new RedirectException(
```

Comment:
Correct, but this shouldn't be part of this PR. I created a separate issue: https://github.com/forkcms/forkcms/issues/1256